pkg/channel/consolidated/reconciler/controller/kafkachannel.go

```diff
@@ -563,6 +582,8 @@ func (r *Reconciler) updateKafkaConfig(ctx context.Context, configMap *corev1.Co
 	// Eventually the previous config should be snapshotted to delete Kafka topics
 	r.kafkaConfig = kafkaConfig
 	r.kafkaConfigError = err
+	configMapDataStr := fmt.Sprintf("%v", configMap.Data)
+	r.kafkaConfigMapHash = fmt.Sprintf("%08x", crc32.ChecksumIEEE([]byte(configMapDataStr)))
```

Comment:
do we really need to stash it here?

how about a `func` that get the cm.Data, and invokes all the lifings:

```
r.kafkaConfigMapHash = provideCmHas()
```

Reply:
can be also tested, as an isolated unit

Reply:
+1, done